docs/components/movement-sensor/_index.md

```diff
@@ -510,10 +510,23 @@ Supported by GPS models.
 
 **Returns:**
 
-- [(Dict[str, float])](https://docs.python.org/3/library/stdtypes.html#str): The accuracy and/or precision of the sensor, if supported.
-  Contents depend on sensor model.
-
-For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/movement_sensor/index.html#viam.components.movement_sensor.MovementSensor.get_accuracy).
+- (MovementSensor.Accuracy): The precision and reliability metrics of the movement sensor, which vary depending on model.
+  This type contains the following fields:
+  - `accuracy` [(map<string, float>)](https://docs.python.org/3/glossary.html#term-mapping): A mapping of specific measurement parameters to their accuracy values.
+    The keys are string identifiers for each measurement (for example, "Hdop", "Vdop"), and the values are their corresponding accuracy levels as floats.
+  - `position_hdop` [(optional float)](https://docs.python.org/3/library/functions.html#float): Horizontal Dilution of Precision (HDOP) value.
+    It indicates the level of accuracy of horizontal measurements.
```

Comment:
```suggestion
    It indicates the level of precision of horizontal measurements.
```
Sorry if I'm missing something but if it's dilution of precision shouldn't this be precision? Looking into it briefly, I see that it can have a knock-on effect on accuracy but from what I'm reading it seems like it is a measure of potential error propagation, not a direct measure of the level of accuracy.

Another option: change this and the following sentence (in all of these) to `"Lower values indicate improved reliability of positional measurements."` or something, since this topic is a little too complex to properly describe in 1-2 sentences anyway, and users should go learn stuff about HDOP before trying to use it meaningfully anyway.

Reply:
I don't know about this, used wording from SME documentation. @susmitaSanyal thoughts?

Reply:
It needs to be called accuracy. I would clarify on it thought. So say something like, 
"It indicates the level of precision of horizontal measurements. Lower values indicate improved reliability of positional measurements."

Reply:
@susmitaSanyal Thanks!

Reply:
Wait actually sorry I'm confused -- you do want to say precision but it needs to be called accuracy?

Reply:
@sguequierre yeah sorry for the confusion. I updated the comment